examples/features/custom_reducers_mnist_pytorch/const.yaml

```diff
@@ -0,0 +1,21 @@
+description: mnist_pytorch_const
```

Comment:
Can you change the name to have custom reducers here in it?

Reply:
done